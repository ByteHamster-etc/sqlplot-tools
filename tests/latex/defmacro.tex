line
%% DEFMACRO
%% SELECT 42.005 AS a, 42.05 AS mc, 42.5 AS abcd, 'hello' AS c
\def\gobble{this}
\def\andthis{...}
line
%% DEFMACRO REFORMAT(round=0)
%% SELECT 42.005 AS a, 42.05 AS mc, 42.5 AS abcd, 'hello' AS c

this is the end