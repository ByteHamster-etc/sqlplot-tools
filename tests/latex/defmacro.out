line
%% DEFMACRO
%% SELECT 42.005 AS a, 42.05 AS mc, 42.5 AS abcd, 'hello' AS c
\def\a{42.005}
\def\mc{42.05}
\def\abcd{42.5}
\def\c{hello}
line
%% DEFMACRO REFORMAT(round=0)
%% SELECT 42.005 AS a, 42.05 AS mc, 42.5 AS abcd, 'hello' AS c
\def\a{42}
\def\mc{42}
\def\abcd{43}
\def\c{hello}

this is the end
